documentation/get-started/tutorials/get-started-qfc.en.md

```diff
@@ -347,6 +347,9 @@ For the old (QGIS) offliner:
 
 !![](../../assets/images/qfc_offline_packager.png,700px)
 
+!!! note
+    This configuration must be set in the Settings section of each project in [QFieldCloud](https://app.qfield.cloud/).
```

Comment:
```suggestion
    This configuration must be set in the Settings page of each project in [QFieldCloud](https://app.qfield.cloud/).
```

Do we call these pages or sections? I usually use pages, but I am ok to call it a section if the documentation already uses this terminology.

Reply:
I completely agree that we should use a page instead of a section.